mmdet/datasets/pipelines/transforms.py

```diff
@@ -1809,3 +1809,271 @@ def __repr__(self):
                      else f'cutout_shape={self.candidates}, ')
         repr_str += f'fill_in={self.fill_in})'
         return repr_str
+
+@PIPELINES.register_module()
+class Mosaic(object):
+    """Mosaic augmentation.
+    (0:cut_y, 0:cut_x)                        -- image 0
```

Comment:
Please refer to RandomCenterCropPad for a more illustrative description.

Reply:
revised